doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -48,7 +48,7 @@ X509_STORE_CTX_verify_fn
 =head1 DESCRIPTION
 
 These functions initialise an B<X509_STORE_CTX> structure for subsequent use
-by X509_verify_cert().
+by L<X509_verify_cert(3)>.
```

Comment:
Please add `or L<X509_STORE_CTX_verify(3)`

Reply:
> Just a few doc nits to fix, otherwise LGTM.

I've handled all of them, 
and in addition extended `X509_STORE_CTX_get_error.pod` accordingly.